assets-src/js/main/translations.js

```diff
@@ -153,23 +153,23 @@ const translate = {
             'team-site': '__team-site'
         },
         'zh-hans': {
-            'admin': '__admin',
-            'backToMainMenu': '__backToMainMenu',
-            'cancelReply': '__cancelReply',
-            'controlsDescription': '__controlsDescription',
-            'logged-in': '__logged-in',
-            'logout': '__logout',
-            'member-site': '__member-site',
-            'menu': '__menu',
-            'my-account': '__my-account',
-            'my-calendar': '__my-calendar',
-            'my-organization': '__my_organization',
-            'nextSlide': '__nextSlide',
-            'previousSlide': '__previousSlide',
-            'sign-in': '__sign-in',
-            'sliderDescription': '__sliderDescription',
-            'slideText': '__Slide {x} of {y}',
-            'team-site': '__team-site'
+            'admin': '管理',
+            'backToMainMenu': '返回主目录',
+            'cancelReply': '取消回复',
+            'controlsDescription': '控件描述',
```

Comment:
```suggestion
            'controlsDescription': '轮播图控件',
```
in English version, it's called **carousel controls**, it's the controller of carousel.